package.json

```diff
@@ -1,11 +1,34 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap",
-    "postinstall": "pip install -e ."
+    "bootstrap": "yarn && yarn conda:forge && yarn dependencies && yarn nbstripout:install && yarn install:dev",
+    "install:dev": "pip install -r requirements-development.txt",
+    "install:prod": "pip install -r requirements.txt",
+    "nbstripout:install": "pip install nbstripout && nbstripout --install",
+    "dependencies": "cross-os dependencies",
+    "dependencies:darwin:win32": "conda install -y --file requirements-dependencies-only.txt",
```

Comment:
A python 3.7 flag here might fix your issue... Or maybe one within the requirements dependencies .txt file?

Reply:
Nope, it can't sit in the requirements file... we could write the python version required in here too... Again... Hmmmm, this WET first DRY later thing is bugging me ...